src/main/kotlin/org/jitsi/nlj/transform/node/incoming/VideoParser.kt

```diff
@@ -55,7 +55,7 @@ class VideoParser(
             return null
         }
         val encodingDesc = findRtpEncodingDesc(videoPacket) ?: run {
-            logger.warn("Unable to find encoding matching packet! encodings=$tracks, packet=$videoPacket")
+            logger.warn("Unable to find encoding matching packet! packet=$videoPacket, encodings=\n${tracks.joinToString(separator = "\n")}")
```

Comment:
Looking at the logs I'm not sure we need/want the newline before printing the encodings?  The current log looks like 
```
JVB 2020-02-20 18:24:58.763 WARNING: [144] [confId=f66d36ab40a12a0e epId=92f86c90 gid=32ab38 stats_id=Jamir-BuO conf_name=shardtest12] VideoParser.findRtpEncodingDesc#84: Unable to find encoding matching packet! packet=RtpPacket: PT=100, Ssrc=4247040970, SeqNum=11279, M=false, X=true, Ts=2770979410, encodings=
MediaStreamTrackDesc 1703982445 has encodings:
  subjective_quality=0,primary_ssrc=2364038633,secondary_ssrcs={948459120=RTX},temporal_id=0,spatial_id=-1
  subjective_quality=1,primary_ssrc=2364038633,secondary_ssrcs={948459120=RTX},temporal_id=1,spatial_id=-1
  subjective_quality=2,primary_ssrc=2364038633,secondary_ssrcs={948459120=RTX},temporal_id=2,spatial_id=-1
  subjective_quality=3,primary_ssrc=833671647,secondary_ssrcs={4114564909=RTX},temporal_id=0,spatial_id=-1
  subjective_quality=4,primary_ssrc=833671647,secondary_ssrcs={4114564909=RTX},temporal_id=1,spatial_id=-1
  subjective_quality=5,primary_ssrc=833671647,secondary_ssrcs={4114564909=RTX},temporal_id=2,spatial_id=-1
  subjective_quality=6,primary_ssrc=1382786376,secondary_ssrcs={1465781216=RTX},temporal_id=0,spatial_id=-1
  subjective_quality=7,primary_ssrc=1382786376,secondary_ssrcs={1465781216=RTX},temporal_id=1,spatial_id=-1
  subjective_quality=8,primary_ssrc=1382786376,secondary_ssrcs={1465781216=RTX},temporal_id=2,spatial_id=-1
```
so I'm not sure the "MediaStreamTrackDesc 1703982445 has encodings:" text needs to be on its own line. But up to you.

Reply:
fixed